src/_guides/language/type-system.md

```diff
@@ -475,13 +476,14 @@ which is disallowed from non-`dynamic` types such as `Animal`.
   in the [analysis options file.][analysis]
 {{site.alert.end}}
 
-To make this code pass static analysis, 
-use an explicit cast, 
-which might fail at runtime.
+To make this type of code pass static analysis, 
+you can use an explicit cast, 
+but it might fail at runtime.
```

Comment:
```suggestion
To make this type of code pass static analysis, 
you can use an explicit cast. 
```
I think the example code should come directly after this line, so the reader can directly pair the syntax they're seeing with the phrase "explicit cast".  Why it might fail still is irrelevant to understanding the code sample.